snowflake_blueprints/upload_file.py

```diff
@@ -116,13 +122,26 @@ def map_snowflake_to_pandas(snowflake_data_types):
     return pandas_dtypes
 
 
+def get_pandas_dates(pandas_datatypes: dict) -> list:
```

Comment:
Aren't there two values being returned? I'm not sure how that's represented in type hinting.

Reply:
Done